test/python/tensorflow/python_tests_list.txt

```diff
@@ -397,6 +397,76 @@ relu_op_test.ReluTest.testGradientFloat32
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.testString
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.testUndefinedIndicesShape
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.testUndefinedOutputShape
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.testUndefinedUpdatesShape
+scatter_nd_ops_test.ScatterNdNonAliasingAddTest.test_session
+scatter_nd_ops_test.ScatterNdTensorTest.testTensorScatterUpdateWithForwarding
+scatter_nd_ops_test.ScatterNdTensorTest.testUpdateAddSub
+scatter_nd_ops_test.ScatterNdTensorTest.testUpdateAddSubGradients
+scatter_nd_ops_test.ScatterNdTensorTest.test_session
+# This test fails on CPU backend on char (and maybe other data types)
+# scatter_nd_ops_test.ScatterNdTest.testBool
+scatter_nd_ops_test.ScatterNdTest.testEmptyOutputShape1
+scatter_nd_ops_test.ScatterNdTest.testEmptyOutputShape2
+scatter_nd_ops_test.ScatterNdTest.testEmptyOutputShape3
+# This test fails on CPU backend on some unsupported data types
+# scatter_nd_ops_test.ScatterNdTest.testExtraIndicesDimensions
+scatter_nd_ops_test.ScatterNdTest.testGradientsRank2ElementUpdate
+scatter_nd_ops_test.ScatterNdTest.testGradientsRank2SliceUpdate
+scatter_nd_ops_test.ScatterNdTest.testGradientsRank3SliceUpdate
+scatter_nd_ops_test.ScatterNdTest.testGradientsRank7SliceUpdate
+scatter_nd_ops_test.ScatterNdTest.testInvalidShape
+scatter_nd_ops_test.ScatterNdTest.testRank3InvalidShape1
+scatter_nd_ops_test.ScatterNdTest.testRank3InvalidShape2
+scatter_nd_ops_test.ScatterNdTest.testRank3ValidShape
+scatter_nd_ops_test.ScatterNdTest.testScatterNdRepatedIndicesAdd
+scatter_nd_ops_test.ScatterNdTest.testSmokeScatterNdBatch1DSliceDim2
+scatter_nd_ops_test.ScatterNdTest.testSmokeScatterNdBatch1DSliceDim3ShapeRank7
+scatter_nd_ops_test.ScatterNdTest.testSmokeScatterNdBatch2DSliceDim2
+scatter_nd_ops_test.ScatterNdTest.testSmokeScatterNdBatch2DSliceDim3ShapeRank7
+scatter_nd_ops_test.ScatterNdTest.testString
+scatter_nd_ops_test.ScatterNdTest.testUndefinedIndicesShape
+scatter_nd_ops_test.ScatterNdTest.testUndefinedOutputShape
+scatter_nd_ops_test.ScatterNdTest.testUndefinedUpdatesShape
+scatter_nd_ops_test.ScatterNdTest.test_session
+scatter_nd_ops_test.StatefulScatterNdTest.testConcurrentUpdates
+scatter_nd_ops_test.StatefulScatterNdTest.testExtraIndicesDimensions
+scatter_nd_ops_test.StatefulScatterNdTest.testRank3InvalidShape1
+scatter_nd_ops_test.StatefulScatterNdTest.testRank3InvalidShape2
+scatter_nd_ops_test.StatefulScatterNdTest.testRank3ValidShape
+scatter_nd_ops_test.StatefulScatterNdTest.testResVarInvalidOutputShape
+scatter_nd_ops_test.StatefulScatterNdTest.testScatterOutOfRangeCpu
+scatter_nd_ops_test.StatefulScatterNdTest.testScatterRepeatIndices
+scatter_nd_ops_test.StatefulScatterNdTest.testSimple
+scatter_nd_ops_test.StatefulScatterNdTest.testSimple2
+scatter_nd_ops_test.StatefulScatterNdTest.testSimple3
+scatter_nd_ops_test.StatefulScatterNdTest.testSimpleResource
+scatter_nd_ops_test.StatefulScatterNdTest.testVariableRankAdd
```

Comment:
these tests were not run before?

Reply:
ScatterND is being added as a new op, so searched for TF python tests that might correspond to scatternd and added them here.